test/verify/check-kdump

```diff
@@ -27,7 +27,7 @@ class KdumpHelpers(MachineCase):
         # all current Fedora/CentOS/RHEL images use BootLoaderSpec
         self.machine.execute("grubby --args=crashkernel=256M --update-kernel=ALL")
         self.machine.execute("mkdir -p /var/crash")
-        self.machine.reboot()
+        self.reboot()
         self.allow_restart_journal_messages()
```

Comment:
No allow_restart_journal messages() call can be removed from all of the tests, when ths follows the new reboot or wait_reboot calls.